app/assets/javascripts/tour/garden.xml.erb

```diff
@@ -0,0 +1,27 @@
+<krpano version="1.0.8.14">
+
+	<view hlookat="0" vlookat="0" maxpixelzoom="1.0" limitview="auto" />
+	<include url="<%= asset_path("tour/skin/defaultskin.xml") %>" />
+
+  <view fisheye="0" limitview="range" hlookatmin="-180" hlookatmax="180" vlookatmin="-90" vlookatmax="90" maxpixelzoom="1.0" fovmax="90" fov="90" hlookat="78" vlookat="10"/>
+  <test url="<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/garden.tiles/preview.jpg" />

```

Comment:
There's no existing helper I can use, but I can use a route helper and replace:

```erb
<test url="<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/garden.tiles/preview.jpg" />
```

with:

```erb
<% url_helpers = Rails.application.routes.url_helpers %>
<test url="<%= url_helpers.public_upload_path("uploaded/number10/garden.tiles/preview", extension: "jpg") %>" />
```

I'm 50:50 on that being better though, thoughts?